src/components/form/form.scss

```diff
@@ -111,3 +111,13 @@
         grid-column: 1/-1;
     }
 }
+
+.form-group {
+    limel-checkbox,
+    limel-switch {
+        margin-left: pxToRem(8);
+        min-height: calc(
+            var(--min-height-of-one-row) - #{pxToRem(15)} /*height of helper text*/
```

Comment:
If we don't have a variable for "height of helper text", maybe we should add one? 🙂

Reply:
great Idea. I'll do a fixup